# Integrative network analysis of transcriptomics data reveals potential prognostic biomarkers for colorectal cancer

**Highlights:**

• To study rewired pathway cross-talk guided by state-dependent gene co-expression \
• State-dependent pathway cross-talk networks were analysed along with domain knowledge \
• PTPRC mediate pathway crosstalk among immune system and with other signaling pathways \
• Cell communication and vascular smooth muscle cross-talk involved in CRC progression \
• Rewired ITGA7 interactions impact the EMT (epithelial-mesenchymal transition) in CRC \

**Datasets:** 

• Gene Expression: Gene Expression Omnibus database (GEO, https://www.ncbi.nlm.nih.gov/geo/) \
• Pathways genesets: MSigDB (Molecular Signature Database, https://www.gsea-msigdb.org/gsea/msigdb) \ 


# Pathway cross-talk
Type I Pathway cross-talk.py and Type II Pathway cross-talk.py Python scripts are designed to identify pathway cross-talk by analyzing co-expressed genes belonging to different pathways. 

# Gene Expression Analysis 
This R script performs the normalization of raw microarray data, identifies differentially expressed genes (DEGs) using the limma package, calculates the Pearson correlation between genes, and compares the correlation patterns across different datasets.

# Gene Regulatory Network analysis 

This R script used to calculate the dysregulation score for each gene based on its change in interaction strength with other genes from the normal state to the CRC state by integrating the gene expression information with the gene regulatory network.



# Author Info
Mohita Mahajan, Angshuman Sarkar and Sukanta Mondal*
Email ID: mahajanmohita93@gmail.com, suku@goa.bits-pilani.ac.in*
